src/main/kotlin/org/malachite/estella/people/api/UserExceptionHandler.kt

```diff
@@ -18,4 +19,8 @@ class UserExceptionHandler {
     @ExceptionHandler(UserNotFoundException::class)
     fun returnBadRequestWhenUserWasntFound(ex: UserNotFoundException) =
```

Comment:
Metoda wstrzykiwana więc trochę bez znaczenia, ale fajnie by było skrócić nazwę, i chyba unikać "return" w nazwie metody xD mogłoby być np "getUserNotFoundResponse()"

Reply:
znaczy ogólnie te nazwy wziąłem z roboty, tam mamy wszędzie `returnBadRequestWhen<sratatata>`, więc mogę to zmienić na luzie, tylko to już masowo będę zmieniał, jak wszystko stąd dogadamy.